db/migrate/20161210134136_create_messages.rb

```diff
@@ -1,7 +1,16 @@
 class CreateMessages < ActiveRecord::Migration[5.0]
+  @@is_sqlite = ActiveRecord::Base.connection.instance_of? ActiveRecord::ConnectionAdapters::SQLite3Adapter
+
   def change
-    create_table :messages do |t|
+    if not @@is_sqlite then
+      execute "CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\""
+    end
```

Comment:
this is the bit that allows the schema to work with sqlite3 and postgresql as well - i'll let it stay - maybe add a comment why it's here